@here/harp-mapview/lib/ElevationRangeSource.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { TilingScheme } from "@here/harp-geoutils";
+import { TileKey } from "@here/harp-geoutils/lib/tiling/TileKey";
+
+export interface ElevationRange {
+    minElevation: number;
+    maxElevation: number;
+}
+
+export enum CalculationStatus {
+    // Calculated approximately. A more precise result may be available later.
+    PendingApproximate,
+    // Calculation completed. The result is final, won't improve upon retrying.
+    FinalPrecise
+}
+
+export interface ElevationRangeSource {
```

Comment:
This will be quite important interface in future, please typedoc at least basic assumptions (whole file).